__tests__/commands/run.js

```diff
@@ -25,13 +25,20 @@ test('lists all available commands with no arguments', (): Promise<void> => {
   return runRun([], {}, 'no-args', (config, reporter): ?Promise<void> => {
     const rprtr = new reporters.BufferReporter({stdout: null, stdin: null});
     const scripts = ['build', 'prestart', 'start'];
+    /* eslint-disable quotes */
+    const hints = {
+      build: "echo 'building'",
+      prestart: "echo 'prestart'",
+      start: "node index.js",
```

Comment:
This one should be single quotes since there is nothing to escape. That is why you were getting the ESLint error.